modules/packages/BLAS.chpl

```diff
@@ -119,57 +119,57 @@ in memory.
-.. _SYMV:   https://software.intel.com/en-us/mkl-developer-reference-fortran-symv
-.. _SYR:    https://software.intel.com/en-us/mkl-developer-reference-fortran-syr
-.. _SYR2:   https://software.intel.com/en-us/mkl-developer-reference-fortran-syr2
-.. _TBMV:   https://software.intel.com/en-us/mkl-developer-reference-fortran-tbmv
-.. _TBSV:   https://software.intel.com/en-us/mkl-developer-reference-fortran-tbsv
-.. _TPMV:   https://software.intel.com/en-us/mkl-developer-reference-fortran-tpmv
-.. _TPSV:   https://software.intel.com/en-us/mkl-developer-reference-fortran-tpsv
-.. _TRMV:   https://software.intel.com/en-us/mkl-developer-reference-fortran-trmv
-.. _TRSV:   https://software.intel.com/en-us/mkl-developer-reference-fortran-trsv
-.. _ASUM:   https://software.intel.com/en-us/mkl-developer-reference-fortran-asum
-.. _AXPY:   https://software.intel.com/en-us/mkl-developer-reference-fortran-axpy
-.. _COPY:   https://software.intel.com/en-us/mkl-developer-reference-fortran-copy
-.. _DOT:    https://software.intel.com/en-us/mkl-developer-reference-fortran-dot
-.. _SDOT:   https://software.intel.com/en-us/mkl-developer-reference-fortran-sdot
-.. _DOTC:   https://software.intel.com/en-us/mkl-developer-reference-fortran-dotc
-.. _DOTU:   https://software.intel.com/en-us/mkl-developer-reference-fortran-dotu
-.. _NRM2:   https://software.intel.com/en-us/mkl-developer-reference-fortran-nrm2
-.. _ROT:    https://software.intel.com/en-us/mkl-developer-reference-fortran-rot
-.. _ROTG:   https://software.intel.com/en-us/mkl-developer-reference-fortran-rotg
-.. _ROTM:   https://software.intel.com/en-us/mkl-developer-reference-fortran-rotm
-.. _ROTMG:  https://software.intel.com/en-us/mkl-developer-reference-fortran-rotmg
-.. _SCAL:   https://software.intel.com/en-us/mkl-developer-reference-fortran-scal
-.. _SWAP:   https://software.intel.com/en-us/mkl-developer-reference-fortran-swap
-.. _AMAX:   https://software.intel.com/en-us/mkl-developer-reference-fortran-i-amax
-.. _AMIN:   https://software.intel.com/en-us/mkl-developer-reference-fortran-amin
-.. _CABS1:  https://software.intel.com/en-us/mkl-developer-reference-fortran-cabs1
+.. _ASUM:   https://www.intel.com/content/www/us/en/develop/documentation/onemkl-developer-reference-fortran/top/blas-and-sparse-blas-routines/blas-routines/blas-level-1-routines/asum.html
+.. _AXPY:   https://www.intel.com/content/www/us/en/develop/documentation/onemkl-developer-reference-fortran/top/blas-and-sparse-blas-routines/blas-routines/blas-level-1-routines/axpy.html
+.. _COPY:   https://www.intel.com/content/www/us/en/develop/documentation/onemkl-developer-reference-fortran/top/blas-and-sparse-blas-routines/blas-routines/blas-level-1-routines/copy.html
+.. _DOT:    https://www.intel.com/content/www/us/en/develop/documentation/onemkl-developer-reference-fortran/top/blas-and-sparse-blas-routines/blas-routines/blas-level-1-routines/dot .html
```

Comment:
@ghbrown : Looks like some of the three-character routines have a stray space in their links?

Reply:
Yes, thanks for catching that. That should be fixed, as well as some other mistakes from my bulk editing.

I've gone through and clicked all the links to make sure they work as of the most recent commit.